admin-dev/themes/default/template/helpers/form/form.tpl

```diff
@@ -34,7 +34,7 @@
 {if isset($identifier_bk) && $identifier_bk == $identifier}{capture name='identifier_count'}{counter name='identifier_count'}{/capture}{/if}
 {assign var='identifier_bk' value=$identifier scope='parent'}
 {if isset($table_bk) && $table_bk == $table}{capture name='table_count'}{counter name='table_count'}{/capture}{/if}
-{assign var='table_bk' value=$table scope='parent'}
+{assign var='table_bk' value=$table scope='root'}
```

Comment:
Reducing the visibility of this variable may break some compatibility.

Reply:
well it actually enhance it since it is visible by root and not only parent
without this the couting can't work because the two forms don't have the same immediate parent